.eslintrc.yml

```diff
@@ -34,7 +36,6 @@ rules:
   eqeqeq: error
   func-call-spacing: error
   func-name-matching: error
-  func-style: error
```

Comment:
How intentional are these rules; is it fine to modify as I see fit or do you feel strongly about them?

I've had success with the [Airbnb linter config](https://github.com/airbnb/javascript) which has become a bit of a standard within React; we could discuss if you're interested.